actionpack/lib/action_dispatch/routing/inspector.rb

```diff
@@ -45,7 +45,11 @@ def action
       end
 
       def internal?
-        controller.to_s =~ %r{\Arails/(info|mailers|welcome)} || path =~ %r{\A#{Rails.application.config.assets.prefix}\z}
+        internal_controller = controller.to_s =~ %r{\arails/(info|mailers|welcome)}
+        internal_asset = Rails.application.config.respond_to?(:assets) &&
```

Comment:
it's defined by `sprockets-rails` now, since https://github.com/rails/rails/commit/5172d9335931559332125d8f50b75d74af295a15